cost/aws/s3_bucket_size/README.md

```diff
@@ -17,7 +17,13 @@ The following policy actions are taken on any resources found to be out of compl
 
 - Send an email report
 
-### Required Permissions
+### Required RightScale Roles
+ 
+- Cloud Management - Actor
+- Cloud Management - Observer
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer